zc_common/jwt_auth/authentication.py

```diff
@@ -13,7 +13,12 @@ class User(object):
     """
 
     def __init__(self, **kwargs):
+        self.pk = kwargs.pop('pk', None) or kwargs.pop('id', None)
+        self.id = self.pk
         self.roles = []
+        self.companies = []
```

Comment:
This array of companies isn't used/necessary here, is it? It looks like everything is handled by the `company_permissions` dictionary which will contain the `id` for the key and `permission` for the value.